src/page/sm-test/publishStreamManagerSocialPusher/index.js

```diff
@@ -205,12 +205,15 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
 	async function newPushIt() {
+		sendButton.disabled = true
+
 		// gather info
+		let smApi = "v1"
 		const {
 			host,
 			app,
 			stream1,
-			streamManagerAPI: version,
+			smApi: version,
 			streamManagerNodeGroup: nodeGroup,
 		} = configuration
```

Comment:
setting `smApi` version here is not doing what you think. The string `v1` will always be its value. The destructuring and reassignment of 

```
streamManagerAPI: version
```

essentially says I have re-assigned the configuration property of `streamManagerAPI` to variable named `version`. That way the stream manager api version can be set in a single configuration and accessed by all tests.

Your change is not reassigning the `let smApi` to anything.